gaas-ui/src/components/view-graphs/view-graphs.tsx

```diff
@@ -115,21 +105,17 @@ export default class ViewGraph extends React.Component<{}, IState> {
                                     <Gauge
                                         maxValue={graphs.length}
                                         data={[
-                                            {key: "TOTAL", data: graphs.length},
+                                            { key: "TOTAL", data: graphs.length },
                                             {
                                                 key: "UP",
-                                                data: graphs.filter((graph) => graph.getStatus() === "UP").length
+                                                data: graphs.filter((graph) => graph.getStatus() === "UP").length,
                                             },
                                             {
                                                 key: "DOWN",
-                                                data: graphs.filter((graph) => graph.getStatus() === "DOWN").length
+                                                data: graphs.filter((graph) => graph.getStatus() === "DOWN").length,
```

Comment:
Does this linter like commas after the last element in the object? Don't think I've seen that before